src/selectors/columnWidths.js

```diff
@@ -41,8 +41,8 @@ let columnDefinition;
  * }} The total width of all columns.
  */
 function columnWidths(columnGroupProps, columnProps, scrollEnabledY, width) {
-  const scrollbarSpace = scrollEnabledY ? Scrollbar.SIZE + Scrollbar.OFFSET : 0;
-  const viewportWidth = width - scrollbarSpace;
+  const scrollbarSpace = scrollEnabledY ? Scrollbar.SIZE : 0;
+  const viewportWidth = width - scrollbarSpace - 1; // Reserve space for last column border
```

Comment:
I think we should only reserve 1px if the scrollbar exists.

So should this be:
```js
  const scrollbarSpace = scrollEnabledY ? Scrollbar.SIZE - 1 : 0; // Reserve space for last column border
  const viewportWidth = width - scrollbarSpace;
```
right?

I think the 1px should be treated as a constant variable. Instead of removing `Scrollbar.OFFSET`, can you add a new variable called `Scrollbar.BORDER_WIDTH` in `src/Scrollbar.js` set to 1px? This will also help in finding it in the future.

Reply:
No, that was the original issue. Space was being reserved differently depending on scrollbar visibility and this change makes the behavior consistent. Let me know if you need more details.